keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -595,6 +589,10 @@ supported container runtime version will fail to run on the node, because the
 For security reasons, volume mounts should set the [`noexec`] and `ro`
 (read-only) options by default.
 
+Note: in the process of mounting images into the container's rootfs, there may need to be intermediate mounts created. This is especially relevant if
```

Comment:
How does this feature play with a separate container runtime filesystem?

Let's say we have 
nodefs (ephemeral-storage + kubelet fs)
imagefs (container runtime filesystem)

It isn't clear to me if this will be mounted on container runtime filesystem or on the kubelet filesystem.

Reply:
This is up to the runtime implementation, the kubelet will not manage the actual path on disk.